Automattic/jetpack
# Custom CSS: add deprecation warning in codebase

## Proposed changes:

The [Custom CSS](https://jetpack.com/support/custom-css/) feature will soon be deprecated from Jetpack.

This PR ensures the codebase issues a deprecation warning when loading the Custom CSS module.

### Other information:


## Jetpack product discussion


pfwV0U-4M-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Download this branch locally and spin up your test site
- Make sure you can see the server logs
- Search for the _Custom CSS_ section on the settings page: `/wp-admin/admin.php?page=jetpack#/settings?term=css`
- Enable the CSS customization panel enhancement
<img width="313" alt="Screenshot 2024-05-02 at 4 05 00 PM" src="https://github.com/Automattic/jetpack/assets/1620183/faa8f27f-e0c2-4a58-86e9-e77c57b81374">

- Visit the Dashboard page: `/wp-admin/admin.php?page=jetpack#/dashboard`
- In the logs, you should see a deprecation warning: `Hook custom_css_loaded is deprecated since version jetpack-$$next-version$$! Use WordPress Custom CSS instead. Jetpack no longer supports Custom CSS...`
